MangoAPI.Client/src/app/components/settings/settings.component.ts

```diff
@@ -163,7 +155,6 @@ export class SettingsComponent implements OnInit {
 
       this.clearProfilePictureFile();
     } catch (e) {
-      this._errorNotificationService.notifyOnError(e);
```

Comment:
This catch block leads to bug, because error already caught in interceptor, so that Image won't be cleared